r/tests/testthat/test-dplyr-mutate.R

```diff
@@ -74,6 +74,16 @@ test_that("transmute", {
   )
 })
 
+test_that("transmute respect bespoke dplyr implementation", {
+  ## see: https://github.com/tidyverse/dplyr/issues/6086
+  compare_dplyr_binding(
+    .input %>%
+      transmute(dbl, int = int + 6L) %>%
+      collect(),
+    tbl
+  )
+})
```

Comment:
Could we add a test with unnamed arguments to `transmute()` here as well?

Reply:
As in an unnamed argument like this?
```r
df |> 
  transmute("foo")
```